locales/en-US/messages.properties

```diff
@@ -166,6 +166,17 @@ homeworkExcuseGeneratorRemixTitle=Remix the {{ homeworkExcuseGenerator }} projec
 homeworkExcuseGeneratorTeachingKitTitle=Read the {{ homeworkExcuseGenerator }} teaching kit
 mySixWordSummerRemixTitle=Remix the {{ mySixWordSummer }} project
 mySixWordSummerTeachingKitTitle=Read the {{ mySixWordSummer }} teaching kit
+lessonPlan=Lesson Plan
+projectBy=By
+remixGalleryTitle=Remix a project to get started...
+projectSingular=project
+projectPlural=projects
+found=found
```

Comment:
If it’s concatenated with a number, we will need proper plural here, too. At least singular/plural depending on the support you have